app/views/games/show.html.erb

```diff
@@ -1,5 +1,10 @@
 <p id="notice"><%= notice %></p>
 
+<div class="white-player-image">
+  <% binding.pry %>
+  <%= image_tag @user.gravatar_url %>
```

Comment:
`@game.white_player.gravatar_url`

hint: a `white_player` is a `User` (see the association).

If it's nil, then the game was not setup correctly. When created, `white_player` should be set to `current_user`